systers_portal/meetup/mixins.py

```diff
@@ -0,0 +1,24 @@
+from django.core.exceptions import ImproperlyConfigured
+
+
+class MeetupLocationMixin(object):
+    """Mixin to add information about MeetupLocation to context, as per the slug"""
+    meetup_location = None
```

Comment:
Since slug will also need to be called using self in mixin, we decided to follow current approach.